### PR TITLE
NSFS | Update noobaa log document and add syslog-ng

### PR DESCRIPTION
**Pull request by naveenpaul1**
### Explain the changes
1. pdate log doc https://github.com/noobaa/noobaa-core/blob/master/docs/NooBaaNonContainerized/Logging.md
2. add syslog-ng basic configuration steps in doc.

### Issues: Fixed #xxx / Gap #xxx
1. https://github.com/noobaa/noobaa-core/issues/8407

### Testing Instructions:
1. 

- [X] Doc added/updated
- [ ] Tests added

### CODE REVIEW

File: docs/NooBaaNonContainerized/Logging.md [hunk 1]
```diff
@@ -19,10 +19,19 @@ This document provides an overview of the logging capabilities within NooBaa, de
 
 ## NooBaa Service Logs
 
-NooBaa service logs are written directly to stderr and to syslog -  
-1. NooBaa writes the logs into stderr and to `/var/log/noobaa.log`.
-2. The `journalctl` service watches stderr and writes it to `journal`.
-3. A `syslog` service, like `rsyslog/syslog_ng` watches `journal` and writes it to `/var/log/messages/`.
+NooBaa service logs are written directly to stderr, syslog, or both.
+1. The `journalctl` service watches stderr and writes into `journal`. That internally updates the `/var/log/message`.
```
Comment (romayalon) [inline docs/NooBaaNonContainerized/Logging.md:23]:
```suggestion
1. The `journalctl` service watches stderr and writes into `journal`. That internally updates the `/var/log/messages`.
```

File: docs/NooBaaNonContainerized/Logging.md [hunk 2]
```diff
@@ -19,10 +19,19 @@ This document provides an overview of the logging capabilities within NooBaa, de
...
+Set `LOG_TO_STDERR_ENABLED` property to `true` to  send Noobaa logs to stderr. Noobaa logs will be send to syslog when the property `LOG_TO_SYSLOG_ENABLED` set to `true`.
+
+Default configurations are:
+```
+LOG_TO_STDERR_ENABLED = true;
+LOG_TO_SYSLOG_ENABLED = false;
+```
 
 ### Journal logs
 1. NooBaa service logs -
```
Comment (romayalon) [inline docs/NooBaaNonContainerized/Logging.md:37]: can you also add here a way to fix `Suppressed 6284127 messages from noobaa.service`
RateLimitInterval=0
RateLimitBurst=0
you can attach a link to the fix mentioned in this link - https://my.f5.com/manage/s/article/K70501143

Comment (naveenpaul1) [reply docs/NooBaaNonContainerized/Logging.md:37]: @romayalon we already have this in our code? de we really need to mention it again?
https://github.com/noobaa/noobaa-core/blob/master/src/deploy/standalone/noobaa_syslog.conf

Comment (romayalon) [reply docs/NooBaaNonContainerized/Logging.md:37]: we have it on the code for syslog conf, we experienced this issue on journal (stderr) for several times

Comment (naveenpaul1) [reply docs/NooBaaNonContainerized/Logging.md:37]: Ok will add it for journal,

**Review by romayalon — APPROVED** [review]

**Pull request merged**